### PR TITLE
Script to ease making a release

### PR DESCRIPTION
**Pull request by Zogg**
Code should be self explanatory.

### CODE REVIEW

File: scripts/make-release.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+fi
+
+echo "#############################"
+echo "These credentials will be used"
+echo "to push git PRs into github."
+echo
+echo "~ Enter your GitHub username:"
+read GITHUB_USERNAME
+echo "~ Enter your GitHub password:"
+read -s GITHUB_PASSWORD
```
Comment (siddharthist) [inline scripts/make-release.sh]: What about people (like me) who use two factor auth?

File: scripts/make-release.sh [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+git checkout -b release/$OLD_RELEASE
+
+git checkout master
+git checkout -b feature/release-$RELEASE
+sed -i "s/\/.*/\/mantl-${RELEASE}/g" .bintray
+git add .bintray
+git commit -m "Auto-Release: ${RELEASE}"
+
+git push https://$GITHUB_USERNAME:$GITHUB_PASSWORD@github.com/asteris-llc/mantl-packaging release/$OLD_RELEASE
+git push https://$GITHUB_USERNAME:$GITHUB_PASSWORD@github.com/asteris-llc/mantl-packaging feature/release-$RELEASE
```
Comment (siddharthist) [inline scripts/make-release.sh]: I have my auth set up via SSH in my global gitconfig. Maybe we can make the protocol configurable, as per my last comment as well?

**Comment by siddharthist** [comment]
This is a general comment and shouldn't hold us back from merging once this is otherwise ready, but I think it's good practice to use [ShellCheck](https://github.com/koalaman/shellcheck) on all shell scripts that we write. Cuts down on subtle errors.

File: scripts/make-release.sh [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+echo "~ Enter your GitHub password:"
+read -s GITHUB_PASSWORD
+
+# Update Mantl's yum repositories
+pushd $MANTL_REPO
+git checkout master
+git checkout -b feature/release-$RELEASE-repo-update
+sed -i "s/\:[^\/]*$/\: ${RELEASE}/g" ./roles/repos/defaults/main.yml
+git add roles/repos/defaults/main.yml
+git commit -m "AUTOMATED RELEASE UPDATE: Mantl yum repo points to $RELEASE packages repo"
```
Comment (siddharthist) [inline scripts/make-release.sh]: No need to capitalize this :)

**Comment by siddharthist** [comment]
@Zogg Just to make sure I understand, this largely
 1. Creates a new bintray repo for mantl-packaging
 2. Uploads all packages to the new repo
 3. Updates Mantl to point to that new repo

Maybe we could have a comment at the top of the file, or some associated documentation to that effect?

File: scripts/make-release.sh (hunk 1 above)
Comment (Zogg) [inline scripts/make-release.sh]: I took the simplest path here. Havent thought much about other auths.

**Comment by Theaxiom** [comment]
I reverted this because we are going to be in a different direction.

**Pull request merged**